docker/001-config.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+set -ex
+
+echo "IPFS PATH: ${IPFS_PATH}"
+
+# We backup old config file
+cp ${IPFS_PATH}/config ${IPFS_PATH}/config_bak
+
+# We inject ResourceMgr JSON object in IPFS config
+# See: https://github.com/ipfs/kubo/blob/master/docs/config.md#swarmresourcemgr
+# We also inject the S3 plugin datastore
+# Important: Make sure your fill out the optionnal parameters $CLUSTER_S3_BUCKET, $CLUSTER_AWS_KEY, $CLUSTER_AWS_SECRET in the cloudformation parameters
+cat ${IPFS_PATH}/config_bak | \
+jq ".Swarm.ResourceMgr.Limits.System = { 
+    Memory: 1073741824, 
+    FD: 1024, 
+    Conns: 1024, 
+    ConnsInbound: 256, 
+    ConnsOutboun: 1024, 
+    Streams: 16384, 
+    StreamsInbound: 4096, 
+    StreamsOutbound: 16384 
+}" | \
```

Comment:
https://github.com/ipfs/kubo/blob/master/docs/changelogs/v0.18.md#improving-libp2p-resource-management-integration

Reply:
Hi @lidel. What are you suggesting?
Should I revert back to the default settings?

Reply:
I would not touch these as they are unrelated to the s3 datastore (these are libp2p limits), the user of the Docker image should be the one tweaking these if needed.